Unfortunately my issue persists:

```
CMake Error at /usr/local/Cellar/cmake/3.14.5/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find LibLUV (missing: LIBLUV_LIBRARY LIBLUV_INCLUDE_DIR)
  (Required is at least version "1.29.1")
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.14.5/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindLibLUV.cmake:34 (find_package_handle_standard_args)
  CMakeLists.txt:388 (find_package)
```

It's also very confusing that it says that lib**l**uv is missing and then mentions that 1.29.1 is required, which is the version of libuv. So what it is missing, libuv or libluv?

FWIW, I often had problems with libvterm being the wrong version when not using bundled libraries, so I have to build like this:

```
USE_BUNDLED_DEPS  := ON

DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_BUSTED=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_GETTEXT=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_GPERF=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_LIBICONV=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_LIBTERMKEY=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_LIBUV=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_LIBVTERM=ON
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_LUA=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_LUAJIT=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_LUAROCKS=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_LUV=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_MSGPACK=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_UNIBILIUM=OFF
DEPS_CMAKE_FLAGS  += -DUSE_BUNDLED_UNIBILIUM=OFF
```

--- a/cmake/FindLibLUV.cmake
+++ b/cmake/FindLibLUV.cmake
@@ -6,7 +6,7 @@
 
 find_package(PkgConfig)
 if (PKG_CONFIG_FOUND)
-  pkg_check_modules(PC_LIBTERMKEY QUIET luv)
+  pkg_check_modules(PC_LIBLUV QUIET luv)
 endif()
 
 set(LIBLUV_DEFINITIONS ${PC_LIBLUV_CFLAGS_OTHER})

